omnigibson/envs/env_base.py

```diff
@@ -42,6 +45,15 @@ def __init__(
             automatic_reset (bool): whether to automatic reset after an episode finishes
             flatten_action_space (bool): whether to flatten the action space as a sinle 1D-array
             flatten_obs_space (bool): whether the observation space should be flattened when generated
+            external_camera_kwargs (list or tuple of dict): List of keyword-specific arguments for external cameras. 
```

Comment:
can we add support for this directly in the env config file? I believe @cgokmen requested this feature. Perhaps you could comment further?